sdks/docker-enclave/supervisord.conf

```diff
@@ -43,15 +43,19 @@ stderr_logfile_maxbytes=0
 # docker
 [program:docker]
 command=/bin/dockerd --iptables=false
+priority=10
```

Comment:
the version of supervisord we use has a `depend_on` setting for dependencies

ref: https://github.com/ochinchina/supervisord?tab=readme-ov-file#supervised-program-settings